bot/cogs/tags.py

```diff
@@ -80,7 +91,6 @@ def _get_suggestions(self, tag_name: str, thresholds: Optional[List[int]] = None
 
     async def _get_tag(self, tag_name: str) -> list:
```

Comment:
```suggestion
    def _get_tag(self, tag_name: str) -> list:
```
Similarly, this is no longer required to be `async`